anafi:
  autonomy:
    ros__parameters:
      fixed_frame: false  # Fixed frame for velocity commands
      flightplan_file: ~/ros2_ws/install/anafi_autonomy/share/anafi_autonomy/missions/test.mavlink  # Absolute path to the FlightPlan file
      followme_mode: 2  # FollowMe mode: 1 = look at the target without moving automatically, 2 = follow the target keeping the same vector, 3 = follow the target keeping the same orientation to its direction, 4 = follow the target as it was held by a leash
      hand_launch: false  # Enable hand launched takeoff
      landing_control: false  # Enable control during landing
      mission_type: 0  # Mission type: 0 = flight plan, 1 = follow me
      takingoff_control: false  # Enable control during takeoff
      world_frame: false  # Yaw in world frame

